mailmerge/api.py

```diff
@@ -49,12 +53,53 @@ def parsemail(raw_message):
     recipients = [x[1] for x in addrs]
     message.__delitem__("bcc")
     message.__setitem__('Date', email.utils.formatdate())
-    text = message.as_string()
     sender = message["from"]
-    return (text, sender, recipients)
 
+    return (message, sender, recipients)
 
-def sendmail(text, sender, recipients, config_filename):
+
+def addattachments(message, template_path):
+    """Add the attachments from the message from the commandline options."""
+    if 'attachments' not in message:
+        return message, 0
+
+    # If the message is not already a multipart message, then make it so
+    if not message.is_multipart():
+        multipart_message = email.mime.multipart.MIMEMultipart()
+        for header_key in message:
+            multipart_message[header_key] = message[header_key]
+        original_text = message.get_payload()
+        assert isinstance(original_text, str)
+        multipart_message.attach(email.mime.text.MIMEText(original_text))
+        message = multipart_message
+
+    attachment_filepaths = [filepath.strip()
```

Comment:
Here's the location I was referring to
> How about using duplicate headers instead of a comma separated list?  You'll need to use `message.get_all()`, I think.

> Ref: https://docs.python.org/2/library/email.message.html#email.message.Message.__getitem__

Reply:
Updated in `bdd6028` as mentioned in https://github.com/awdeorio/mailmerge/pull/32#issuecomment-452550888